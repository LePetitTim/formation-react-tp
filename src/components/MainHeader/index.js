import React from 'react';

export const MainHeader = () => <h1>Formation React</h1>;

export default MainHeader;
