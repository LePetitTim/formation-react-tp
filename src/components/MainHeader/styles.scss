.main-header {
  &__nav {
    a {
      display: inline-block;
      padding: .5rem 1rem;
      text-decoration: none;
      color: inherit;
      border-radius: 4px 4px 0 0;

      &.active {
        background: white;
      }
    }
  }
}
