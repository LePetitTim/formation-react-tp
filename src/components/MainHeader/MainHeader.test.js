import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import MainHeader from './';

it('should render', () => {
  const tree = renderer.create(<BrowserRouter><MainHeader /></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
