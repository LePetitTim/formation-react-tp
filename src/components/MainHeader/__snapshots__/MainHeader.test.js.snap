// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="main-header"
>
  <h1>
    Formation React
  </h1>
  <nav
    className="main-header__nav"
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
      style={Object {}}
    >
      Home
    </a>
    <a
      aria-current={null}
      href="/list"
      onClick={[Function]}
    >
      Liste
    </a>
    <a
      aria-current={null}
      href="/create"
      onClick={[Function]}
    >
      Nouvel item
    </a>
  </nav>
</div>
`;
